File: App/MainApp/UiRibbon/Tabs/AppTab.vb
Option Strict On
Option Infer Off
Option Explicit On

Imports Autodesk.Windows

Namespace UiRibbon.Tabs
    Public Class AppTab
        Private Shared ReadOnly tabName As String = "AppTab"

        Public Shared Function CreateAppTab() As RibbonTab
            Dim ribbonTab As RibbonTab = New RibbonTab()
            ribbonTab.Title = tabName
            ribbonTab.Id = tabName
            ribbonTab.Name = tabName
            Return ribbonTab
        End Function
    End Class
End Namespace

File: CSharpApp/MainApp/UiRibbon/Tabs/AppTab.cs

using Autodesk.Windows;

namespace MainApp.UiRibbon.Tabs
{
    public class AppTab
    {
        private static readonly string tabName = "AppTab";

        public static RibbonTab CreateAppTab()
        {
            var ribbonTab = new RibbonTab();
            ribbonTab.Title = tabName;
            ribbonTab.Id = tabName;
            ribbonTab.Name = tabName;
            return ribbonTab;
        }
    }
}